### PR TITLE
Dailymotion Bid Adaptor: initial release

### PR DESCRIPTION
**Pull request by kvnsw**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter : docs PR (prebid.github.io) https://github.com/dailymotion-oss/prebid.github.io/pull/2
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

A new bidder adapter dailymotionBidAdapter added. 

- contact email of the adapter’s maintainer : **ad-leo-engineering@dailymotion.com**

- Test parameters for validating bids:

By setting the following configuration options, you'll get a constant response to any request to validate your adapter integration:

```
config.setConfig({
  dailymotion: {
    api_key: 'dailymotion-testing',
    position: 'preroll'
  }
});
```

```
const adUnits = [{
  code: 'midroll',
  mediaTypes: {
    video: {
      context: 'instream',
      playerSize: [1280, 720],
    },
  },
  bids: [{
    bidder: "dailymotion",
    params: {}
  }]
}];
```

Please note that failing to set these configuration options will result in the adapter not bidding at all.

### CODE REVIEW

**Review by Aditi0101 — CHANGES_REQUESTED** [review]
Please do not merge : We are still waiting for email and prebid service changes.

File: modules/dailymotionBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  },
+
+  /**
+   * Make a server request from the list of valid BidRequests (that already passed the isBidRequestValid call)
+   *
+   * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+   * @param {BidderRequest} bidderRequest
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: (validBidRequests, bidderRequest) => (validBidRequests || []).map(bid => ({
```
Comment (jeremy-le-dev) [inline modules/dailymotionBidAdapter.js]: We could have something like that `buildRequests: (validBidRequests = [], bidderRequest) => validBidRequests.map(bid => ({` but as you did it works 👍🏻

**Review by jeremy-le-dev — APPROVED** [review]

File: modules/dailymotionBidAdapter.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    position: 'test_position',
+    xid: 'x123456'
+  }
+});
+```
+
+This call must be made before each auction. Here's a description of each parameter:
+* `api_key` is your publisher API key. For testing purpose, you can use "dailymotion-testing".
+* `position` parameter is the ad position in the video and must either be "preroll", "midroll" or "postroll".
+* `xid` is the video XID and should be provided to have better contextual data and higher fillrate
```
Comment (kvnsw) [inline modules/dailymotionBidAdapter.md]: Same as https://github.com/dailymotion-oss/prebid.github.io/pull/1/files#r1457209383

File: modules/dailymotionBidAdapter.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+# Sample video AdUnit
+
+```
+const adUnits = [
+  {
+    code: 'test-ad-unit',
+    mediaTypes: {
+      video: {
+        context: 'instream',
+        playerSize: [640, 480],
```
Comment (kvnsw) [inline modules/dailymotionBidAdapter.md]: Same as https://github.com/dailymotion-oss/prebid.github.io/pull/1/commits/db85372c781719c528be270cef961fd03c48424d

File: modules/dailymotionBidAdapter.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Overview
+
+```
+Module Name: Dailymotion Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: ad-leo-engineering@dailymotion.com
```
Comment (Aditi0101) [inline modules/dailymotionBidAdapter.md:6]: We need to update email here as well.

File: modules/dailymotionBidAdapter.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Overview
+
+```
+Module Name: Dailymotion Bid Adapter
```
Comment (jeremy-le-dev) [inline modules/dailymotionBidAdapter.md:4]: "Dailymotion **Bid** Adapter" or "Dailymotion **Prebid** Adapter" ? 
In the description we have "prebid"

Comment (Aditi0101) [reply modules/dailymotionBidAdapter.md:4]: Dailymotion Bid Adapter is ok. Other bid adapter also has similar name.

File: modules/dailymotionBidAdapter.md (hunk 4 above)
Comment (Aditi0101) [reply modules/dailymotionBidAdapter.md:6]: We want to keep Maintainer: ad-leo-engineering@dailymotion.com. so LGTM.

**Review by Aditi0101 — APPROVED** [review]

**Review by jeremy-le-dev — APPROVED** [review]

**Review by sebmil-daily — APPROVED** [review]

**Pull request merged**